### DISCUSSION TITLE
Acronym deciphering

### DISCUSSION THREAD

**Discussion by npajkovsky** [kudrykv/latex-yearly-planner#120] [Q&A] [answered]
Hello,

I'm having a problem to under what files do.

rm2.mos -> Remarkable 2 and mos stands for?
rm2_ddvk -> ddvk stands for?

rm2_ddvk_lh -> ddvk for left-handed?

**Comment by kudrykv** [comment, marked as answer]
MOS — months on side
DDVK is a nickname of a folk that tampers with Remarkable software https://github.com/ddvk/remarkable-hacks
lh — left-handed, yes

**Reply by npajkovsky** [reply]
Thanks.
